src/ShapeCrawler/ShapeCollection/ISpacing.cs

```diff
@@ -18,6 +19,16 @@ public interface ISpacing
     ///     Gets the number of points if Line Spacing specified in points, otherwise <see langword="null"/>. 
     /// </summary>
     double? LineSpacingPoints { get; }
+    
+    /// <summary>
+    ///    Gets or sets the number of points before the paragraph, otherwise <see langword="null"/>.
+    /// </summary>
+    double? BeforeSpacingPoints { get; set; }
```

Comment:
I think `BeforeSpacingPoints`  and `AfterSpacingPoints` should not be nullable. In this case they should return default `0` as PowerPoint Application does it:

![image](https://github.com/user-attachments/assets/ca23ac66-01bd-4da2-8ad4-8b5c665e497b)